packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-container.jsx

```diff
@@ -146,7 +146,7 @@ const ProofOfIdentityContainer = ({
                     setIsLoading(false);
                 });
         });
-    }, []); // eslint-disable-line react-hooks/exhaustive-deps
+    }, [is_loading]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
This is a `componentDidMount` hook which has an empty array. If you make this a dependent hook with `is_loading` in the array. It will send a request whenever `is_loading` changes, i.e., there will be a request when `is_loading` is **true** and another after you set `is_loading` to **false** on line 146. That's an unnecessary request you're sending. Please find another way to resolve this.